bin-src/lib/getDependentStoryFiles.ts

```diff
@@ -64,6 +103,8 @@ export async function getDependentStoryFiles(
     untraced = [],
   } = ctx.options;
 
+  const changedModules = await getExternalChangedDirectDependencies(changedFiles);
+
```

Comment:
As per conversation with @ghengeveld I'll supply this as a parameter to `getDependentStoryFiles` for easier testing.